.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,52 @@
+name: actionlint
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/*.yml'
+      - '.github/workflows/*.yaml'
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+      - '.github/workflows/*.yaml'
+  merge_group:
+
+env:
+  HOMEBREW_DEVELOPER: 1
```

Comment:
```suggestion
env:
  HOMEBREW_DEVELOPER: 1
  HOMEBREW_NO_AUTO_UPDATE: 1
```

Reply:
Worth checking it's not already set by default? This may also be a good reason to use the default GitHub Actions images here and elsewhere: I think they have at least `shellcheck` installed by default.

Reply:
I see `brew update --autoupdate` being run, so it's not set by default.

We could use the default GitHub Actions image, but we'll need to drop `setup-homebrew`, because it takes ages to run in the default image. Let me experiment with it.

Reply:
> We could use the default GitHub Actions image, but we'll need to drop setup-homebrew, because it takes ages to run in the default image.

This is only true in homebrew-core itself. We don't bother cloning it when run under any other repository.